webapp/javascript/redux/actions.js

```diff
@@ -158,6 +167,28 @@ export const setRightFile = (file, flamebearer) => ({
   payload: { file, flamebearer },
 });
 
+export const requestProfiles = () => ({ type: REQUEST_PROFILES, payload: {} });
```

Comment:
I would suggest to qualify these actions a bit more, "requesting profiles" is too broad, maybe "request adhoc profiles"?

Reply:
I specialized all the adhoc related actions at https://github.com/pyroscope-io/pyroscope/pull/624/commits/8eb4b96bcff07204009d9fc46383bc8442edc088